app/Config/Mimes.php

```diff
@@ -12,6 +12,9 @@
  * the most common one should be first in the array to aid the guess*
  * methods. The same applies when more than one mime-type exists for a
  * single extension.
+ *
+ * When working with mime types, please make sure you have the ´fileinfo´
+ * extension enabled to reliably detect the media types.
```

Comment:
For composer installed apps, maybe we can add `ext-fileinfo: For better mime detection` in the suggests section of composer.json

Reply:
I added the suggest section and a notice. Please check, I'm not really comfortable with composer yet 😅

Reply:
Talking about `composer` 😆 : 
Whenever I run `composer update` in my local repo, GitHub finds changes for all files in `system/ThirdParty/Kint`. Is that desired behavior?

Reply:
I am not aware of any updates to kint. Perhaps the copy of kint in your system/ThirdParty is not updated?

Reply:
There is actually no update, the files do not have content changes, but the modification date was changed.
I think its because of the `post-update` script which copies the files for `kint` and `laminas-escaper` even if there was no update to them.

Reply:
Btw. automatically installing the pre-commit hook after `composer-update` does not seem to work on Win, too, because `bash` is not a valid command.

Reply:
Yes, that date modification is intended as `ComposerUpdate` has no way of telling if the copied files are actually changed. It simply copies from `vendor` to `system/ThirdParty`.

I installed my Git executable on Win10 manually using the available Git for Windows installer. Choose from their the option to include install the optional Linux tools and commands. This way I can run `bash -c 'cp env .env'` directly on powershell or cmd.

Reply:
Okay.
Thanks, I think I remember I saw that option, but opted out...